packages/proxy/lib/http/index.ts

```diff
@@ -43,7 +44,10 @@ type HttpMiddlewareCtx<T> = {
   debug: Debug.Debugger
   middleware: HttpMiddlewareStacks
   deferSourceMapRewrite: (opts: { js: string, url: string }) => string
+  getCurrentBrowser: () => Browser | Partial<Browser> & Pick<Browser, 'family'> | null
```

Comment:
my apologies for all the changes you had to make to this file... in retrospect, this was an unmaintainable way to set up how `ctx` works. this area needs a refactor, like, yesterday. not related to this PR, just sharing my shame :smile: